### ISSUE TITLE
How to set font color for chart tooltip and menu?

### ISSUE THREAD

**Issue by sam-wheat** [apexcharts/Blazor-ApexCharts#372]
I use ApexCharts with MudBlazor.  When I use the dark theme the font color for the chart tooltip is white on white background as is the font color for the hamburger menu.  How do I change that color?  Have seen [this doc](https://apexcharts.com/docs/options/tooltip/#).
Thanks.

**Comment by digitaldirk** [comment]
I use this and MudBlazor as well.

To fix the tooltip text color use Mud components inside ApexPointTooltip (More info: [https://apexcharts.github.io/Blazor-ApexCharts/features/tooltip](https://apexcharts.github.io/Blazor-ApexCharts/features/tooltip)) 
```
<ApexPointTooltip>
  <MudPaper Class="pa-2">
    <MudText Typo="Typo.h6">Your data here</MudText>
  </MudPaper>
</ApexPointTooltip>
```
I haven't figured out the hamburger menu yet but for example you can set the axis style like (more info: [https://apexcharts.github.io/Blazor-ApexCharts/features/axis](https://apexcharts.github.io/Blazor-ApexCharts/features/axis)):
```
private AxisLabelStyle chartAxisLabelStyle
{
  get => new AxisLabelStyle
  {
    Colors = new ApexCharts.Color(new List<string> { (ThemeService._isDarkMode ? "white" : "black") })
  };
}
```
and then do something like:
```
options.Xaxis.Labels.Style = chartAxisLabelStyle;
```
or just something like for the legend:
```
options.Legend.Labels.Colors = ThemeService._isDarkMode ? "white" : "black";
```

**Comment by sam-wheat** [comment]
@digitaldirk Thank you.  Based on your comment I discovered the entire chart can be placed in `MudPaper` which fixes both tooltip and menu:

    <MudPaper Style="color:black;background-color:white;">
        <ApexChart>...</ApexChart>
    </MudPaper>

**Comment by digitaldirk** [comment]
**For future reference my final solution is:**

I wrapped the chart in a MudPaper with dark/light style:
```razor
<MudPaper Style="@(ThemeService._isDarkMode ? "color:white;background-color:black" : "color:black;background-color:white")">
  <ApexChart...
```
Then I still had to do the label/legend styles like in my first comment (when in dark mode, the text was hard to read, but by using the option styling the text looks a lot better).

I couldn't get the download hamburger menu to have a dark background in dark mode, for my purposes I just hid it. I was able to set the text color via the MudPaper styling though (so white background with a different text color).

I customized the tooltip as I did in my first comment

**Result:**
![image](https://github.com/apexcharts/Blazor-ApexCharts/assets/22691956/8b96af0c-7a9c-4b11-9a3e-e6c379377064)

![image](https://github.com/apexcharts/Blazor-ApexCharts/assets/22691956/e7b5d48c-182a-4e8a-9ee6-4367bbb45520)
